README.md

```diff
@@ -12,29 +12,52 @@
 
 <!--te-->
 
-# Particle.One Data API client
+# Particle.One Data API
 
 ## Description
 
-This is user-friendly wrapper for P1 Data REST API
+This package contains:
+- A Python module `p1_data_client_python` wrapping the P1 Data REST API into a
+  pandas-friendly interface
+- An example notebook on how to use the Python module
+  `notebook/p1_data_client_example.ipynb`
+
+- An example Jupyter notebook on how to connect to the P1 Data REST API directly:
+  `notebooks/p1_data_api_v1_example.ipynb`
 
 ## Prerequisites
 
-* Python version >= 3.7
+- Python version >= 3.7
 
 ## Installation
 
-```shell script
-pip install p1_data_client_python
-```
+- To install from PyPI:
+  ```shell script
+  > pip install p1_data_client_python
```

Comment:
I would use `p1-data-client-python` (dashes instead of underscores) for the official name in PyPI.
But it's your call.